### PR TITLE
#87429 Fix "Who's Attending?" List Reporting

### PR DESCRIPTION
**Pull request by ggwicz**
_Ref:_ [C#87429](http://central.tri.be/issues/87429)

### CODE REVIEW

File: src/Tribe/Tickets_View.php [hunk 1]
```diff
@@ -357,7 +357,18 @@ public function intercept_template( $old_file, $template ) {
 	 */
 	public function inject_link_template() {
 		$event_id = get_the_ID();
-		$user_id = get_current_user_id();
+		$user_id  = get_current_user_id();
+
+		/**
+		 * A flag we can set via filter, e.g. at the end of this method, to ensure this template only shows once.
+		 *
+		 * @since TBD
```
Comment (borkweb) [inline src/Tribe/Tickets_View.php]: Add `@param boolean $already_rendered`

File: readme.txt [hunk 2]
```diff
@@ -233,6 +233,10 @@ The plugin is produced by [Modern Tribe Inc](http://m.tri.be/18uc).
 
 == Changelog ==
 
+= [4.5.6] TBD =
+
+* Fix - Prevent multiple instances of the "View your RSVPs and Tickets" link from showing on single events (or other ticket-enabled post types). [87429]
```
Comment (borkweb) [inline readme.txt]: Add props?

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for **, but this was pull requested against *release/M17.18*.

**Comment by tr1b0t** [comment]
Closing this Pull Request because it is pull requested against the wrong branch. The Central ticket is for **, but this was pull requested against *release/M17.18*.

**Pull request merged**